### DISCUSSION TITLE
Can't join discord

### DISCUSSION THREAD

**Discussion by thewh1teagle** [pykeio/ort#251] [General]
I can't join the discord group because it force me to verify my phone and discord have some issue with that.
Please downgrade the verification level so people can join it

https://www.iorad.com/player/2103047/Discord---How-to-change-your-server-verification-level#trysteps-1

**Comment by thewh1teagle** [comment]
![image](https://github.com/user-attachments/assets/29737371-9514-4deb-b65a-639eb73d033e)

**Comment by thewh1teagle** [comment]
@decahedron1
Why closing it? I still can't join it. Is is that important to have that requirement to join the group?

**Reply by decahedron1** [reply]
Yes, the server's had an issue with spam in the past. The verification level was lowered to medium (requiring a verified email) a few days ago so I'm not sure why you'd still be unable to join.

**Reply by thewh1teagle** [reply]
Finally I could join. thanks.

By the way I understand that a lot of people ask dumb questions here and you don't have time sometimes but even answer in 3 words and closing & locking is better then just closing....
